README.md

```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
-
-* generate-script (--repos-only)
-* migrate-repo
-* grant/revoke-migrator-role
-* create-team
-* add-team-to-repo
-* configure-autolink
-
-#### I want to do the above and also lock & disable the repository being migrated from Azure DevOps
-In order to use the following pre & post migration commands:
-
-* lock-ado-repo
-* disable-ado-repo
-* generate script (without the --repos-only flag)
-
-You will need to include these additional scopes for your Azure DevOps personal access token in addition to the ones listed in GEI's [documentation](https://docs.github.com/en/early-access/github/migrating-with-github-enterprise-importer/migrating-to-github-enterprise-cloud-with-the-importer#step-2-assign-the-migration-permissions-role-and-ensure-the-migrator-has-the-expected-pat-scopes):
-
-* `Service Connection (Read)`
-* `Build (Read & execute)`
-* `Security (Manage)`
-* `Code (Read, write, and manage)`
-
-#### I want to do the above and also re-connect Azure Pipelines & Boards to the newly migrated repository on GitHub
-If you want to re-connect an Azure Pipline or Board to the migrated repo then you'll need your ADO personal access token to be `full access`.
-
-![image](https://user-images.githubusercontent.com/40493721/145903240-6a6d04cd-ba03-47f4-84aa-6af741a8ddd6.png)
-
-At this point, you can now begin to run and test the import process.
+## Quick Start Videos
+You'll find videos below to help you quickly get started with the GEI CLI. Be sure to pick the videos relevant to your migration scenario. 
 
-#### Run the Migrations
+>NOTE: We don't update these videos as often as we update the CLI, so they may not exactly match the functionality in the latest release of this CLI.
 
-Once you have configured the `octoshift`(*gh-gei*) command-line tool and `environment variables` for the person access tokens, you can run the command-line tool and see all available options.
-![octoshift help](https://user-images.githubusercontent.com/29484535/145065029-ea8b3fcd-fcea-4f9b-ba7e-f9d3407e17fa.png)
+### Migrating GitHub to GitHub
+The quick start video below will help you start migrating between GitHub organizations. 
 
-The first step you will want to run is `generate-script` to help outline all commands to migrate an entire **Azure Project**
-This command will generate a `migrate.ps1` file in the local folder. You will want to open it with an editor tool as it can be quite large. It's recommended that you include the `--repos-only` flag the first time. 
-Tools like `Atom`, `VSCode`, or `NotePad++` are great ways to see the data.
+* Migrating GitHub to GitHub with the GEI CLI: https://youtu.be/5cQkM_8n5YY
```

Comment:
We need to remove this video as it mentions using `.sh` extensions.

Reply:
done, I plan to re-record all videos hopefully next week